components/Table/Table.js

```diff
@@ -271,6 +274,33 @@ export class Table {
     }
   }
 
+  renderModalWindow(matchObj, rowId) {
+    const modalWindowContainer = document.createElement('div');
+    modalWindowContainer.classList.add('modal-window-container');
+    const bgShadow = document.createElement('div');
+    bgShadow.classList.add('bg-shadow');
+    this.tableContainer.append(modalWindowContainer, bgShadow);
+    bgShadow.addEventListener('click', () => {
+      modalWindowContainer.remove();
+      bgShadow.remove();
+    });
```

Comment:
This should be some method, like `remove` in the `Modal` that will clean DOM and make other needed actions